cpp/src/IceGrid/Internal.ice

```diff
@@ -387,8 +387,8 @@ class InternalReplicaInfo
 
 interface InternalRegistry extends FileReader
 {
-    /// Register a node with the registry. If a node with the same name is already registered, [registerNode] will
-    /// overide the previous node only if it's not active.
+    /// Register a node with the registry. If a node with the same name is already registered,
+    /// <code>registerNode</code> will overide the previous node only if it's not active.
```

Comment:
Since you're updating this doc comment, may as well fix formatting and spelling mistake too. E.g.

If a node with the same name is already registered, this operation overrides the existing registration only when the previously registered node is not active.